.vscode/launch.json

```diff
@@ -0,0 +1,20 @@
+{
```

Comment:
Perhaps we should add .vscode to .gitignore file, as it can contain developer specific settings..

Reply:
Okey, I added to .gitignore.
Do you prefer that we close this pull request and that I make several small pull requests?

Reply:
Do you prefer that we close this pull request and that I make two small pull requests?
- adding naming conventions
- adding tags & service factory & api factory & naming for tags